### PR TITLE
Tech stack overview

### PR DESCRIPTION
**Pull request by DarkPurple141**
Comment with your thoughts.

Stack options:
* None, just write raw html, css, js. 
* Vue.js or React for prerendered Static
* Django, Rust, Express or Flask for basic web app
* All of the above for REST + Vue/React for frontend
* React/Vue with Express for full SSR.

### CODE REVIEW

File: docs/technology.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+![SSR](https://cdn-images-1.medium.com/max/2000/1*jJkEQpgZ8waQ5P-W5lhxuQ.png)
+
+![SSR](https://cloud.githubusercontent.com/assets/499550/17607895/786a415a-5fee-11e6-9c11-45a2cfdf085c.png)
+
+Highest order of application complexity but also closest to real world scenario. This is quite opinionated about technology, you basically need a node server OR a server able to spawn a node process to run the server side render, then send what's called 'hydrating' data to the client.
+
+## Summary
+
+One thing about all of these is there is plenty of decent boiler-plate out thereso initial setup and what not isn't that complex, necessarily.
```
Comment (NunoDasNeves) [inline docs/technology.md]: Typo: thereso

Comment (PapaPwn) [reply docs/technology.md]: Important pick up! Good work!

**Comment by DarkPurple141** [comment]
@PapaPwn I agree, likely the easiest, but you miss all the fancy SPA things for that full web app immersion. But maybe not a biggie.

**Comment by Aruzal** [comment]
I'm currently interested in playing around with either Django or NodeJS REST + Vue/React. Although, we should probably look more at what features we want to add and then base a decision off that?

**Comment by DarkPurple141** [comment]
@Aruzal Yes I'd prefer REST (any) + Frontend (any) as it's much more realistic modern architecture even if slightly more work. But yep. Maybe do another wednesday dinner discussion.

**Comment by DarkPurple141** [comment]
https://github.com/teaching-vue/vue-demo This one? Else just read the docs. Don't mind on React or Vue. Should really have this discussion on Slack ayyy.

**Pull request merged**